app/src/main/java/org/mozilla/fenix/utils/ToolbarPopupWindow.kt

```diff
@@ -34,6 +34,9 @@ object ToolbarPopupWindow {
         val context = view.get()?.context ?: return
         val isCustomTabSession = customTabSession != null
```

Comment:
super nit picky: we could move this assignment below the new return, save a few cpu cycles if it's not needed.